actions/utility.js

```diff
@@ -0,0 +1,28 @@
+import { camelToSnake } from '../utility';
+
+export const getUrl = (playerId, options, url) =>
+  `${url(playerId)}${options.reduce((previous, current) =>
+    current.values.reduce((total, value) => `${previous}${current.queryParam}=${value}&`, ''), '?')}`;
+
+export const addQueryString = options => Object.keys(options).map(key => ({
```

Comment:
I think we can add this but it would be a refactoring effort since our data structure isn't shaped like this { prop: value, prop2: [array, of, values] }. I can keep this module in mind though and see if there is an opportunity to refactor that at some point